tests/test_device.py

```diff
@@ -694,3 +708,129 @@ def test_outdated_API(self, monkeypatch):
             m.setattr(qml, "version", lambda: "0.0.1")
             with pytest.raises(DeviceError, match="plugin requires PennyLane versions"):
                 qml.device("default.qubit", wires=0)
+
+
+class TestBatchExecution:
+    """Tests for the batch_execute method."""
+
+    def test_results_no_interface(self, tol):
+        """Tests that the correct results are computed using tapes without an interface."""
+        # skip test if interface cannot be imported
+
+        qml.enable_tape()
```

Comment:
Is this needed? I think this is only needed if we want to create a tape-mode QNode.

Reply:
Oh sorry!